src/pcre2_intmodedep.h

```diff
@@ -471,7 +471,7 @@ code. */
 /* These are trivial for the 32-bit library, since all UTF-32 characters fit
 into one PCRE2_UCHAR unit. */
 
-#define MAX_UTF_SINGLE_CU (0x10ffffu)
+#define MAX_UTF_SINGLE_CU 0x10ffffu
```

Comment:
This and the previously similar change are somehow unrelated, but unlikely to cause problems